app/src/services/api.ts

```diff
@@ -87,6 +94,9 @@ export const openclimateAPI = createApi({
     getOCCity: builder.query<any, string>({
       query: (q) => `/api/v1/search/actor?q=${q}`,
     }),
+    getOCCityData: builder.query<any, string>({
```

Comment:
Instead of any here you could use {locode: string} (same for the others).
This means you have type safety when using this hook in a component.

Reply:
Nvm, the response is marked as any. Would be nice to add the type definition for that though, so the usage of API data can be properly type checked.